subworkflows/local/call_sv_manta.nf

```diff
@@ -0,0 +1,43 @@
+//
+// A structural variant caller workflow for manta
+//
+
+include { MANTA_GERMLINE as MANTA } from '../../modules/local/manta/germline/main'
+
+workflow CALL_SV_MANTA {
+    take:
+    bam            // channel: [ val(meta), path(bam) ]
+    bai            // channel: [ val(meta), path(bai) ]
+    fasta          // path(fasta)
+    fai            // path(fai)
+    ch_case_info   // channel: [ case_id ]
+    ch_bed         // channel: [ val(meta), path(bed), path(bed_tbi) ]
+
+    main:
+        bam.collect{it[1]}
+            .toList()
+            .set { bam_file_list }
+
+        bai.collect{it[1]}
+            .set { bai_file_list }
+
+        ch_case_info.combine(bam_file_list)
+            .set { manta_input_bams }
+
+        if (params.analysis_type == "WGS") {
+            MANTA ( manta_input_bams, bai_file_list, fasta, fai, [[],[],[]] )
+        } else {
+            ch_target_bed = ch_bed.ifEmpty([[],[],[]])
+            MANTA ( manta_input_bams, bai_file_list, fasta, fai, ch_target_bed )
```

Comment:
Just a comment that we can adress in a later PR. As the model is written if WES has been set and no target bed file has been given, manta will be executed in WGS mode. There should be check to see that a target file has been given if wes mode is active. However that check shouldn't be here.

Reply:
ah yes! Good catch. First had WES there, and then I changed it to WGS to test if the loop was working or not. Forgot to change it back to WES 🤦🏻

Reply:
Good point! I'll do that in a different PR.